.github/workflows/helm-lint-and-test.yaml

```diff
@@ -0,0 +1,27 @@
+name: helm
+
+on:
+  pull_request:
+    paths:
+      - "charts/perses/**"
+    branches:
+      - main
+
+jobs:
+  helm-lint-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: setup
+        uses: ./.github/actions/setup_helmlint
+        with:
+          create-kind-cluster: "true"
+      - name: run chart-testing (install)
+        run: ct install --charts charts/perses
+      - name: install mdox
+        run: go install github.com/bwplotka/mdox@latest
+      - name: run chart README lint
+        run: make --always-make update-helm-readme && make --always-make fmt-docs && make --always-make checkdocs
```

Comment:
running simply `make` without the option `always-make` should work don't you think ?

Reply:
Ops!
It was a copy and paste from my terminal hehe